styles/variables/_layout.scss

```diff
@@ -1,8 +1,2 @@
 // Base rhythm units for layout and spacing (margins, padding)
 $space: 1rem;
-$space--xxs: $space * (1/4);
-$space--xs: $space * (1/2);
-$space--s: $space * (3/4);
-$space--m: $space;
-$space--l: $space * 2;
-$space--xl: $space * 4;
```

Comment:
You didn't think they were useful? Or did you think it was going to be a lot of importing?
Expressions like this ones `$-space * 0.75` feel a little bit arbitrary. On the other hand `$-space--s` feels more codified.